code/modules/mob/language/station.dm

```diff
@@ -92,12 +92,12 @@
 
 /datum/language/russian
 	name = LANGUAGE_CYRILLIC
-	desc = "Ancient language of Russian colonists, rusted with time and bastardized with technical terms in everyday use."
-	colour = "russian"
+	desc = "A language derived from Earth's Eastern European and Central Asian regions. Commonly spoken in forntier colonies and finding its home on many agricultural worlds."
```

Comment:
probably want that last sentence to be

"Commonly spoken in frontier colonies, it finds its home on many agricultural worlds."